lib/metalsmith.js

```diff
@@ -10,7 +10,7 @@ const tagcleaner = require('metalsmith-tagcleaner')    // Use tag cleaner to rem
 const permalinks = require('metalsmith-permalinks')    // apply a permalink pattern to files
 const sass = require('metalsmith-sass')                // convert Sass files to CSS using LibSass
 
-const debug = require('./debug')                       // debug plugin
+// const debug = require('./debug')                    // debug plugin
```

Comment:
If it's not being used can we remove this line?

Reply:
I left this so people can uncomment and call the `debug()` plugin on [line 168](https://github.com/alphagov/govuk-design-system/blob/master/lib/metalsmith.js#L168) to output the metalsmith main objects (files and metadata).